datadog_checks_base/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ***Changed***:
 
+* Upgrade `psycopg2-binary` to `v2.9.8` ([#15949](https://github.com/DataDog/integrations-core/pull/15949))
```

Comment:
No need for an entry here. I understand CI will fail, you can add the no changelog label to fix that

```suggestion
```